.github/workflows/vale.yml

```
@@ -5,6 +5,7 @@ on:
     paths:
       - 'guides/common/**.adoc'
       - '.github/workflows/vale.yml'
+      - '.vale/styles/config/**.txt'
```

How about just triggering on any vale change?
```suggestion
      - '.vale/**'
      - .vale.ini
```

---

But this is only when we trigger the job. IMHO anything in `.vale` affects how Vale behaves so we should trigger CI on it. There's no reason to be more specific. `.vale/**` should still trigger when `.vale/styles/config/vocabularies/Foreman/accept.txt` changes.

---

OK, you might have a point. I have never tested this. I will make the change and see what happens.